### PR TITLE
[IMPROVEMENT] Sanitize Install.sh Using Shellcheck

### PR DESCRIPTION
**Pull request by thealphadollar**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [X] I am an active contributor to the project.

---

The PR sanitizes the current install.sh and implements contemporary methods of handling shell scripts. 

[ShellCheck](https://www.shellcheck.net/) was used to modernise the code.

Pep8 was also added to ensure local sanity check for Python.

NOTE: There are reports of unused variables but due to unholistic knowledge, they were not removed.

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
Can you split off the PR?

1 for the shellcheck change, 1 for the other?

**Comment by thealphadollar** [comment]
> Can you split off the PR?
> 
> 1 for the shellcheck change, 1 for the other?

Done. I've kept only install.sh changes here, and creating a new PR for pep8

File: install/install.sh [hunk 1]
```diff
@@ -28,10 +28,13 @@ echo "* Updating package list"
 apt-get update >> "$install_log" 2>&1
 echo "* Installing nginx, python, pip, kvm, libvirt and virt-manager"
 apt-get -q -y install nginx python python-dev python3-libvirt libxslt1-dev libxml2-dev python-pip qemu-kvm libvirt-bin virt-manager mediainfo >> "$install_log" 2>&1
-if [ ! -f /etc/init.d/mysql* ]; then
-    echo "* Installing MySQL (root password will be empty!)"
-    DEBIAN_FRONTEND=noninteractive apt-get install -y mysql-server >> "$install_log" 2>&1
-fi
+for file in /etc/init.d/mysql*
```
Comment (canihavesomecoffee) [inline install/install.sh]: Will this loop run if there's no files at all that are matching this pattern?

Comment (thealphadollar) [reply install/install.sh]: Yes, it works. I've tested for that :smile:

**Pull request merged**